### DISCUSSION TITLE
Selected option is not checked

### DISCUSSION THREAD

**Discussion by bozhenova** [megahertz/react-simple-wysiwyg#25] [General]
Hey! Thanks for you lib! It's super simple yet sufficient for most of the tasks.
Is it intended behaviour that when I select style in Styles the selected option is not checked because of this line https://github.com/megahertz/react-simple-wysiwyg/blob/HEAD/src/toolbar/dropdowns.tsx#L36 ? Or is it a typo?

**Comment by megahertz** [comment]
Thanks for the report, fixed

**Reply by bozhenova** [reply]
Wow, that was quick! Thank you :)
